stern/tail.go

```diff
@@ -99,6 +103,30 @@ func (o TailOptions) IsInclude(msg string) bool {
 	return false
 }
 
+var colorHighlight = color.New(color.FgRed, color.Bold).SprintFunc()
+
+func (o TailOptions) HighlightMatchedString(msg string) string {
+	if o.reHightlight == nil {
```

Comment:
```suggestion
	if len(o.Include) == 0 {
		return msg
	}
	
	if o.reHightlight == nil {
```

If the include option is not specified, the following process is unnecessary, so I prefer to add an early return.